docs/blog/2018-10-04-journey-to-the-content-mesh/index.md

```diff
@@ -0,0 +1,62 @@
+# Delivering Modern Website Experiences: The Journey to a Content Mesh
+
+The CMS was born to make content sites _possible_. Now, it’s being reimagined to make them _incredible_.
+
+In the early 2000s, publishing on the web was difficult. The CMS emerged as a single application to store content, build sites, and deliver them to users.
+
+Over time, tools in each category got better and more specialized, while the quality of monolithic enterprise CMS applications like Adobe Experience Manager and Sitecore has stayed roughly the same.
+
+Today, a website team running an e-commerce site can store product inventory in [Shopify](https://www.shopify.com/), product listings in [Salsify](https://www.salsify.com/) and reviews in [Bazaarvoice](https://www.bazaarvoice.com/). A team running a paywalled content site can create stories in [Wordpress](https://wordpress.org/), store video in [JWPlayer](https://www.jwplayer.com/), user data in [Auth0](https://auth0.com/), and subscription data in [Recurly](https://recurly.com/).
+
+When website teams want to add search, they can turn to [Algolia](https://www.algolia.com/); for payments, [Stripe](http://stripe.com); for analytics, [Segment](https://segment.com/); for A/B testing, [Optimizely](https://www.optimizely.com/); for personalization, [Evergage](https://www.evergage.com/).
+
+Teams are adopting these services because each is tailored to a specific use-case, while the generic enterprise CMS alternative is optimized for none.
```

Comment:
Amazing first few paragraphs! Really drew me in

Reply:
>while the generic enterprise CMS alternative is optimized for none.

This feels a bit too straw-man-ish. There's plenty of enterprise CMSs that claim to optimize for one use case or another. Perhaps close the sentence by saying something like the CMS is increasingly just the place where different services are integrated together — which of course then easily leads people to wonder whether the CMS is a great fit for that role & what value are they getting from their CMS exactly.

Reply:
The important piece here is to draw sharp distinctions. I'll soften this a bit though.